runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -246,6 +246,13 @@
 	private static final String TEMPOROSS_LOOT_STRING = "You found some loot: ";
 	private static final int TEMPOROSS_REGION = 12588;
 
+	// Camdozaal Vault lockbox
+	private static final Map<Integer, String> LOCKBOX_IDS = new ImmutableMap.Builder<Integer, String>().
+		put(ItemID.SIMPLE_LOCKBOX_25647, "Simple lockbox").
+		put(ItemID.ELABORATE_LOCKBOX_25649, "Elaborate lockbox").
+		put(ItemID.ORNATE_LOCKBOX_25651, "Ornate lockbox").
+		build();
```

Comment:
This can be stored as an immutable list of item IDs, and the event name can be set using `MenuOptionClicked#getMenuTarget()` instead of doing this lookup. As-is, this doesn't serve any purpose.

Reply:
while this normally is true, for items it seems that getMenuTarget() is actually just the option too.